### ISSUE TITLE
Configure presets externally

### ISSUE THREAD

**Issue by vsevel** [quarkusio/code.quarkus.io#1480]
**Description**

Presets are configured in `PlatformService.PRESETS`. I would like to instantiate my own `code.quarkus.io`, with my own presets. For instance: small mycompany serverless app, medium mycompany app, mycompany batch processing app, ...

Currently, if I had to configure my own presets, I would need to fork the project, which I do not want to do.

Since the application does not have any database, and we probably do not want to make it stateful at this point, a possible solution would be to accept some SR configuration to configure additional presets:

```
presets:
  db-service-reactive:
    title: Microservice with database
...
  mycompany-serverless-app
    title: ...
```

**Comment by ia3andy** [comment]
yeah that was just the first step goal is to have them in the platform data
